rever/activities/bibtex.xsh

```diff
@@ -0,0 +1,76 @@
+"""Activity for bumping version."""
+import datetime
+import os
+
+from rever import vcsutils
+from rever.activity import Activity
+try:
+    import bibtexparser
+except ImportError:
+    bibtexparser = None
+
+
+def render_authors(authors):
+    """Parse a list of of tuples of authors into valid bibtex
+
+    Parameters
+    ----------
+    authors: list of str
+        The authors eg ['Your name in nicely formatted bibtex'].
+        Please see ``<http://nwalsh.com/tex/texhelp/bibtx-23.html>`` for
+        information about how to format your name for bibtex
+
+    Returns
+    -------
+    str:
+        Valid bibtex authors
+    """
+    if len(authors) == 1:
+        return ''.join(authors[0])
```

Comment:
This still seems incorrect.  Could you write a unit test for this function please that covers the following cases: 0 authors, 1 author, 2+ authors.  Thanks!

Reply:
I might just steal the author stuff from regolith.